package/MDAnalysis/analysis/contacts.py

```diff
@@ -254,7 +254,7 @@ def soft_cut_q(r, r0, beta=5.0, lambda_constant=1.8):
     Q : float
       fraction of native contacts
 
-    References
+    """ References
     ----------
     .. [Best2013] RB Best, G Hummer, and WA Eaton, "Native contacts determine protein
        folding mechanisms in atomistic simulations" _PNAS_ **110** (2013),
```

Comment:
The idea is to have this and all the references included in MDAnalysis in BibTeX format. References in the User Guide are in BibTex. See [this link](https://github.com/MDAnalysis/UserGuide/blob/develop/doc/source/references.bib#L2) as a hint.

Reply:
I have created a bibtex file using Texmaker. I need help getting it in the repository. I was considering citing the repository, do you think that would solve it?

Reply:
I'm not sure what you mean by citing the repository. We don't need to here,  we are citing Best and Franklin. You also seem to have added your file into the repository already?